### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by SKR301**
created doc and updated readme

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+**DSICLAIMER:** *To the few lifeless human beings who actually thought to visit and read this page, This documentation may contain some harsh language. The owner isn't responsible for your feelings at the end of this journey. I'm wasting way too much time in this stupid thing, I hope you put the package in some good use.*
```
Comment (norkator) [inline docs.md:1]: remove this maybe? People don't really care and understand these.

File: docs.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+# Metric-Clock
+A clock to tell time in metric(10) system. **v1.1.0**
+<br /><br />
+*Ahh! here we go again...hate writing docs. No one is gonna read it anyway, and start raising issues and complaining. Ughhh. Anyway...*
```
Comment (norkator) [inline docs.md:6]: keep it professional

File: docs.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+	);
+}
+```
+
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143640604-2953de75-a30e-48fc-aa8f-8a8b0f6b8bad.png" alt="n2mDate"/>
+</p>
+
+*For anyone wondering why I chose 30-nov and not 14-May! Well that's none of your business...but 30th nov is my birthday*
```
Comment (norkator) [inline docs.md:236]: "that's none of your business" please -> professionalism, I got feeling you are attacking me and now I am PISSED OFF like John Wick

File: docs.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143627652-48ce66d0-4b39-48f9-b290-39a52ed3428e.png" alt="m2nTime"/>
+</p>
+
+*You get the gist, right? You can figure out the other functions...*<br />
+*What?? You can't! BOO-HOO-HOO I'm a developer still I don't know how to figure small things out.*<br />
+*Did I hurt your petty little feeligns there buddy? Oh my bad...But sorry this ain't google which just shout out billion ways to centre a div.*<br />
+
```
Comment (norkator) [inline docs.md:179]: (ಠ_ರ) ?

**Review by norkator — CHANGES_REQUESTED** [review]
This doc gives me negative hype. Doc instructions are fine but up to you if you want to clean that extra content away.

File: docs.md (hunk 4 above)
Comment (SKR301) [reply docs.md:179]: Noted, will remove all personal notes!

File: docs.md (hunk 2 above)
Comment (SKR301) [reply docs.md]: made it "professional"

**Review by norkator — APPROVED** [review]

**Pull request merged**